t_server_options/test_server_options.py

```diff
@@ -56,6 +62,182 @@ class TestServerOptions(tester.TempestaTest):
 """,
     }
 
+
+class TestRescheduling(TestServerOptionsBase):
```

Comment:
t_server_options tests was disabled on remote setup before. Is this expected for this class?